src/components/AddSong/styles.ts

```diff
@@ -0,0 +1,70 @@
+import styled from 'styled-components'
+
+export const Wrapper = styled.div`
+    margin: auto;
```

Comment:
what about delete all the margin:auto and replace with align-items?

Reply:
fixed.